pkg/iac/terraform/state/test/aws_appautoscaling_target/terraform.tf

```diff
@@ -0,0 +1,61 @@
+provider "aws" {
```

Comment:
This is not very usefull to put the tf file for the IaC test, the state file in json is enough, if needed we can mutate it manually